### PR TITLE
Popout recent events

### PR DESCRIPTION
**Pull request by andycreeth**

### CODE REVIEW

File: app/components/pages/Live.vue.ts [hunk 1]
```diff
@@ -26,16 +27,33 @@ export default class Live extends Vue {
...
 
   mounted() {
     this.i18nService.setWebviewLocale(this.$refs.webview);
+
+    this.$refs.webview.addEventListener('new-window', e => {
+      const match = e.url.match(/dashboard\/([^\/^\?]*)/);
+
+      if (match && match[1] === 'recent-events') {
+        this.popout();
+      } else if (match) {
```
Comment (holiber) [inline app/components/pages/Live.vue.ts]: I'm a bit curious in which case we should redirect here?

File: app/components/windows/RecentEvents.vue [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<template>
+<modal-layout
+  title="Recent Events"
```
Comment (holiber) [inline app/components/windows/RecentEvents.vue]: $t('Recent Events') here

File: app/components/pages/Live.vue.ts (hunk 1 above)
Comment (andycreeth) [reply app/components/pages/Live.vue.ts]: There are a couple buttons in the recent events feed that link to the spin wheel and credits widget settings.  Before this change they have just never worked.  I saw an opportunity to make them start doing the correct thing pretty easily in the context of this larger change.

File: app/components/windows/RecentEvents.vue (hunk 2 above)
Comment (andycreeth) [reply app/components/windows/RecentEvents.vue]: Thanks.  I need to start thinking about translations when I add UI now.

Comment (andycreeth) [reply app/components/windows/RecentEvents.vue]: Do I need to add this to one of the JSON files?

Comment (holiber) [reply app/components/windows/RecentEvents.vue]: Yeah if it is not present there

**Pull request merged**